jdaviz/configs/default/aida.py

```diff
@@ -102,23 +124,20 @@ def set_viewport(self, center=None, fov=None, image_label=None, **kwargs):
         fov : `~astropy.units.Quantity` or tuple of floats
             Set the width of the viewport to span `field_of_view`.
 
-        image_label : str
             Set the viewport with respect to the image
             with the data label: ``image_label``.
         """
-
-        image, image_label = self._get_image_glue_data(image_label)
```

Comment:
This removed line should be kept or moved into `_set_fov`. If no `image_label` is given, `_get_image_glue_data` will grab the top image layer in jdaviz, and we should preserve that.